spec/features/venues/create_new_venue_spec.rb

```diff
@@ -20,5 +20,7 @@
     expect(page).to have_content(venue['website'])
     expect(page).to have_content(venue['address'])
     expect(page).to have_content(t_is_accessible_string)
+
+    expect(page).to have_content('Venue saved successfully')
   end
```

Comment:
This code should not be part of this pull request. I think this is another one of those branching from a branch issues.